transactions/node-version-beacon/2023/apr-24/NodeVersionBeacon.cdc

```diff
@@ -0,0 +1,518 @@
+/// The NodeVersionBeacon contract holds the past and future protocol versions.
```

Comment:
this was pulled from here: https://github.com/onflow/flow-core-contracts/blob/master/contracts/NodeVersionBeacon.cdc